### PR TITLE
Remove files

### PR DESCRIPTION
**Pull request by sammyCofactory**
Remove main.py and update bot examples.

### CODE REVIEW

File: docs/docs/examples/huggingface-bot.md
Comment (bharatchhabraai) [inline docs/docs/examples/huggingface-bot.md:1]: Why docs/docs/examples?

Just have this /examples/hugging-face-bot/main.py and  /examples/openai-bot/main.py

Comment (sammyCofactory) [reply docs/docs/examples/huggingface-bot.md:1]: docs/docs/examples are the example text files which are present in the docs hosted at docs.textbase.ai.

Comment (bharatchhabraai) [reply docs/docs/examples/huggingface-bot.md:1]: ah..ok. atleast move examples into their own folders

Comment (sammyCofactory) [reply docs/docs/examples/huggingface-bot.md:1]: Done

Comment (bharatchhabraai) [reply docs/docs/examples/huggingface-bot.md:1]: name them `main.py`

Comment (sammyCofactory) [reply docs/docs/examples/huggingface-bot.md:1]: Done

**Review by bharatchhabraai — APPROVED** [review]

**Pull request merged**